documentation/meta/incidents/index.md

```diff
@@ -0,0 +1,154 @@
+should be marked as "critical" in GitHub. Other issues should be prioritized at
+your own discretion.
+
+- **Severity 1**: Critical issue affecting all users and all functionalities of
+  Openverse.
+- **Severity 2**: Major issue affecting a large number of users or critical
+  functionality of Openverse.
+- **Severity 3**: Minor issue affecting a limited number of users or
+  non-critical functionality of Openverse.
+
+### Status
+
+- **Stabilization pending** - The incident is actively disrupting service
+- **Stabilized** - The incident is no longer directly disrupting service. root
+  causes may
+- **Under investigation** - The root causes of the incident are being
+  researched. Stabilized but not yet resolved
+- **Resolved** - The incident is stable, and long-term fixes to prevent future
+  occurrences or other mitigations have been identified.
+
+As you might expect, incidents can be _destabilized_ if they reoccur, and the
+state should rollback to "Stabilization pending":
+
+```{mermaid}
+graph LR
+    A[Stabilization pending]
+    B[Stabilized]
+    C[Under investigation]
+    D[Resolved]
+
+    A --> B
+    B --> C
+    B --> A
+    C --> A
+    C --> D
+    D --> A
+```
+
+## How do we _handle_ incidents?
+
+```{warning}
+Dealing with an incident now? Go to the [Incident Response Runbook](/meta/incidents/runbooks/incident_response_runbook).
+```
+
+This section addresses the philosophy of our approach to incident management and
+explanation behind the steps in the
+[runbook](/meta/incidents/runbooks/incident_response_runbook).
+
+There are four main steps to handling production incidents:
+
+<!-- prettier-ignore -->
+| # | Name | Goal | Status After |
+|---|------|------|--------------|
+| 1 | Record | Visibly acknowledge the incident and create the communication threads necessary for ongoing discussion and investigation. Understand how bad the situation is based on our severity definitions. | Stabilization pending OR Stabilized (as appropriate)
+| 2 | Stabilize |<p>If there is a total or partial outage, identify service stabilization options and action them. | Stabilized |
+| 3 | Resolve | Identify outstanding improvements to the service that would mitigate the issue in the future and create tickets to implement. | Under investigation |
+| 4 | Review | Identify process improvements. If the underlying cause is still unknown, identify further steps to resolve or continue investigation of the issue via 5-whys. | Under investigation OR Resolved (as appropriate) |
+
+All maintainers are required to report incidents they encounter. **Crucially,
+maintainers who record incidents are not _inherently_ responsible for
+stabilizing them.** Their only required job is to record the incident. Every
+incident has a **Lead** who is the person responsible for providing updates on
+the incident and delegating tasks. The Lead is also responsible for hosting the
+retrospective after the incident is resolved.
```

Comment:
This is a very helpful distinction and mostly covers the concern I raised above about needing to write multiple incident reports in a row for e.g., severity 3 things.

I'll suggest one small change for clarity:

```suggestion
All maintainers are required to record incidents they encounter. **Crucially,
maintainers who record incidents are not _inherently_ responsible for
stabilizing them.** Their only required job is to record the incident. Every
incident has a **Lead** who is the person responsible for providing updates on
the incident and delegating tasks. The Lead is also responsible for hosting the
retrospective after the incident is resolved.
```

Using "record" as the verb rather than "report" when we mean "the first and only necessary immediate step for all incidents regardless of severity" would help with consistency and clarity of the precise expectation.